doc/examples/tutorial_null_distributions.md

```diff
@@ -0,0 +1,286 @@
+tuning_curves_smooth.name = "firing rate"
+tuning_curves_smooth.attrs["unit"] = "Hz"
+tuning_curves_smooth.coords["X"].attrs["unit"] = "cm"
+tuning_curves_smooth.coords["Y"].attrs["unit"] = "cm"
+tuning_curves
+```
+
+We can further also compute our metric of interest.
+In this case, we are interested in the modulation of the neurons' firing by position. 
+A metric that is typically used to quantify this is the mutual information, often called [spatial information](https://proceedings.neurips.cc/paper/1992/hash/5dd9db5e033da9c6fb5ba83c7a7ebea9-Abstract.html) for this case specifically:
+```{code-cell} ipython3
+spatial_information = nap.compute_mutual_information(tuning_curves)
+spatial_information
+```
+
+We can visualize all that together:
+```{code-cell} ipython3
+g = tuning_curves_smooth.plot(col="unit")
+for ax, (unit_idx, unit_id) in zip(g.axs.flat, enumerate(units)):
+    ax.set_title(f"unit={unit_id}\nSI={spatial_information['bits/spike'][unit_idx]:.2f}")
+plt.show()
+```
+Some of these units seem to have clear spatial firing (e.g. unit 3), with localized firing fields and relatively high spatial information values.
+However, for units like unit 0 we can not say for certain that the pattern we see is not just caused by random activity.
+
+***
+Randomization
+-------------
+Next, we can start thinking about null distributions.
+Our ultimate goal is to be able to say that a neuron's activity is significantly modulated by position.
+To do that, we want to contrast it to the activity of a neuron that is not modulated by position at all.
+But how do we pick that neuron? 
+We do not want to stray away from the firing patterns of our neuron of interest too much, so we can't just pick something arbitrary.
+
+In practice, we will often take the activity of our neuron of interest and shuffle it somehow, thereby breaking the relation with our variable of interest (position), but keeping its general firing statistics as much as possible.
+
+Pynapple provides four methods for such shuffling:
+- [`jitter_timestamps`](pynapple.process.randomize.jitter_timestamps): jitters timestamps independently by random amounts uniformly drawn from a given range.
+- [`resample_timestamps`](pynapple.process.randomize.resample_timestamps): uniformly resamples all timestamps within the time support.
+- [`shift_timestamps`](pynapple.process.randomize.shift_timestamps): shifts all timestamps by a random amount drawn from a given range, wrapping the end of the time support to the beginning.
+- [`shuffle_ts_intervals`](pynapple.process.randomize.shuffle_ts_intervals): randomizes timestamps by shuffling the intervals between them.
+
+Let's apply them to an example neuron to look at their effect on spatial firing:
+```{code-cell} ipython3
+jitter = nap.jitter_timestamps(units[3], max_jitter=2.0)
+resample = nap.resample_timestamps(units[3])
+shift = nap.shift_timestamps(units[3], min_shift=10.0)
+interval_shuffle = nap.shuffle_ts_intervals(units[3])
```

Comment:
Is it possible to have a cell for this? Or break those 4 lines into 4 difference cells? That would probably make it more clear what is happening.